src/middlewared/middlewared/api/base/types/user.py

```diff
@@ -23,7 +23,7 @@
 
 DEFAULT_VALID_CHARS = string.ascii_letters + string.digits + '_' + '-' + '.'
 DEFAULT_VALID_START = string.ascii_letters + '_'
-DEFAULT_MAX_LENGTH = 32
+DEFAULT_MAX_LENGTH = 32  # WARNING UT_NAMESIZE = 32. If we go above this then utmp accounting may break
```

Comment:
While you're in here... line 32 below has a extra space between `valid_start_chars` and the colon.